content/applications/services/helpdesk.rst

```diff
@@ -238,6 +238,23 @@ Select a stage to fold temporarily, then click the :guilabel:`⚙️ (gear)` ico
    Manually folding a stage from the Kanban view is temporary and does **not** close the tickets in
    the stage.
 
+Merge tickets
+=============
+
+If duplicate tickets are found in *Helpdesk*, they can be combined into a single ticket using the
+*merge* feature.
+
+.. important::
+   The *merge* feature is **only** accessible if the :doc:`Data Cleaning
+   <../productivity/data_cleaning>` application is installed on the database.
+
+To merge two or more tickets, navigate to :menuselection:`Helpdesk app --> Tickets --> All Tickets`.
+Identify the tickets to be merged and tick the checkbox at the far left of each ticket. Then, click
+the :icon:`fa-cog` :guilabel:`Actions` icon, and select :guilabel:`Merge` from the drop-down menu.
+Doing so opens a new page where the selected tickets are listed with their :guilabel:`Similarity`
+rating. From here, tickets can be :ref:`merged <data_cleaning/merge-records>` or
+:guilabel:`Discarded`, as desired.
```

Comment:
```suggestion
Doing so opens a new page where the selected tickets are listed with their :guilabel:`Similarity`
rating. From here, click either :ref:`Merge <data_cleaning/merge-records>` to combine the tickets, or
:guilabel:`DISCARD`.
```

Only suggesting this so you can have the GUI's with the text as it appears on the screen.